packages/stream-tests/src/__tests__/multiquery.test.ts

```diff
@@ -0,0 +1,285 @@
+import tmp from 'tmp-promise'
+import type { Ceramic } from '../../../core/src/ceramic.js'
+import { createCeramic } from '../../../core/src/__tests__/create-ceramic.js'
+import { TileDocument } from '@ceramicnetwork/stream-tile'
+import { IpfsApi, SyncOptions, TestUtils } from '@ceramicnetwork/common'
+import { StreamID } from '@ceramicnetwork/streamid'
+import { createIPFS } from '@ceramicnetwork/ipfs-daemon'
+import getPort from 'get-port'
+import { CeramicDaemon, DaemonConfig } from '@ceramicnetwork/cli'
+import { CeramicClient } from '@ceramicnetwork/http-client'
+import { jest } from '@jest/globals'
+describe('multiquery API http-client tests', () => {
+  let ipfs: IpfsApi
+  let ceramic: CeramicClient
+  let core: Ceramic
+  let daemon: CeramicDaemon
+  let tmpFolder: tmp.DirectoryResult
+  let streamA: TileDocument,
+    streamB: TileDocument,
+    streamC: TileDocument,
+    streamD: TileDocument,
+    streamE: TileDocument,
+    streamF: TileDocument
+  const notExistStreamId = StreamID.fromString(
+    'kjzl6cwe1jw1495fyn7770ujykvl1f8sskbzsevlux062ajragz9hp3akdqbmdg'
+  )
+  const streamFTimestamps = []
+  const streamFStates = []
+
+  beforeAll(async () => {
+    ipfs = await createIPFS()
+    tmpFolder = await tmp.dir({ unsafeCleanup: true })
+    const port = await getPort()
+    const apiUrl = 'http://localhost:' + port
+    ceramic = await createCeramic(ipfs, {
+      stateStoreDirectory: tmpFolder.path,
+      networkName: 'inmemory',
+    })
+    core = await createCeramic(ipfs)
+    daemon = new CeramicDaemon(
+      core,
+      DaemonConfig.fromObject({
+        'http-api': { port: port, 'admin-dids': [core.did.id.toString()] },
+        node: {},
+      })
+    )
+    await daemon.listen()
+    ceramic = new CeramicClient(apiUrl)
+    ceramic.did = core.did
+
+    streamF = await TileDocument.create(ceramic, { test: '321f' })
+    streamE = await TileDocument.create(ceramic, { f: streamF.id.toUrl() })
```

Comment:
only streamF is ever used in tests, no need to make these other streams.

Reply:
Actually no need to make streamF here globally either, you can just make a new stream at the beginning of each test.